tools/link/src/utils/install.ts

```diff
@@ -1,7 +1,13 @@
+import { getPackageManager } from '@lg-tools/meta';
+import { SupportedPackageManager } from '@lg-tools/meta/src/getPackageManager';
 import { spawn } from 'cross-spawn';
+import fsx from 'fs-extra';
 
+/**
+ * @deprecated use `installPackages` instead
+ */
 export function yarnInstall(path: string) {
-  return new Promise((resolve, reject) => {
+  return new Promise(resolve => {
```

Comment:
should we just rm this instead of deprecating it?

Reply:
Probably 👍